src/Client/ApiKey.php

```diff
@@ -77,4 +77,147 @@ public function getCurrent(): ResultApiKey
             throw $this->getExceptionByStatusCode($method, $url, $response);
         }
     }
+
+    /**
+     * Create a new API key for current user.
+     *
+     * @param string $label Visible label on API key overview
+     * @param array $permissions The permissions array can contain specific store id
+     * e.g. btcpay.server.canmanageusers:2KxSpc9V5zDWfUbvgYiZuAfka4wUhGF96F75Ao8y4zHP
+     */
+    public function createApikey(?string $label = null, ?array $permissions = null): ResultApiKey
+    {
+        $url = $this->getApiUrl() . 'api-keys';
+        $headers = $this->getRequestHeaders();
+        $method = 'POST';
+
+        $body = json_encode(
+            [
+                'label' => $label,
+                'permissions' => $permissions
+            ],
+            JSON_THROW_ON_ERROR
+        );
+
+        $response = $this->getHttpClient()->request($method, $url, $headers, $body);
+
+        if ($response->getStatus() === 200) {
+            return new ResultApiKey(json_decode($response->getBody(), true, 512, JSON_THROW_ON_ERROR));
+        } else {
+            throw $this->getExceptionByStatusCode($method, $url, $response);
+        }
+    }
+
+     /**
+     * Create a new API Key using Basic Auth. You need to pass email and password of the user.
+     */
+    public function createApiKeyBasicAuth($email, $password, ?string $label = null, ?array $permissions = null): ResultApiKey
```

Comment:
This function and the `getRequestHeadersBasicAuth` function was reverted again after I merged because this can now be done without basic auth and we decided that we do not want to keep that functionality in the codebase to have only the api key way to do stuff. 

See this revert of a previous merge: https://github.com/btcpayserver/btcpayserver-greenfield-php/pull/101

Reply:
Ah yes, forgot about that. Done.